README.md

```diff
@@ -31,20 +31,24 @@ If you want to run your own instance of the bot the following packages are requi
 * nodejs >=10.16.0
 * npm
 * postgres >=11.5
+* node-pg-migrate
+* pg
 
-Make sure to create the gatekeeper database.
+Make sure to create the gatekeeper database and run the migration to create the tables.
 
 ```
 postgres=# CREATE DATABASE gatekeeper;
 CREATE DATABASE
 postgres=# \q
 ```
 
+
 ```bash
 git clone https://github.com/Kruhlmann/gatekeeper
 cd gatekeeper
 npm install
 npm run upgrade
+DATABASE_URL=postgres://<user>:<password>@localhost:5432/<name> npm run migrate up
```

Comment:
This should be `npm run upgrade` since the database URL is constructed from the environment variables in there.

Reply:
How about that, i'm dumb.